### PR TITLE
Disable autocomplete.js if `WHOOGLE_AUTOCOMPLETE=0`

### PR DESCRIPTION
**Pull request by DUOLabs333**

### CODE REVIEW

**Comment by DUOLabs333** [comment]
Should fix #610

**Review by benbusby — CHANGES_REQUESTED** [review]
The treatment of this environment variable needs to be inverted. `WHOOGLE_AUTOCOMPLETE` should only ever used to disable the feature. I'd rather not have an entire feature suddenly disappear for the majority of users who are accustomed to it being enabled by default.

**Comment by DUOLabs333** [comment]
But it is -- if it doesn't exist, it is 1 by default.

**Comment by benbusby** [comment]
Ah I see. The title of the PR threw me off and I hadn't looked at the code much yet. Will review now.

File: app/routes.py [hunk 1]
```diff
@@ -37,6 +37,8 @@
 newest_version = '' if current_version >= newest_version \
     else newest_version
 
+ac_var='WHOOGLE_AUTOCOMPLETE'
+autocomplete_enabled=os.getenv(ac_var,'1')
```
Comment (benbusby) [inline app/routes.py:41]: pep8 fixes:

```suggestion
ac_var = 'WHOOGLE_AUTOCOMPLETE'
autocomplete_enabled = os.getenv(ac_var, '1')
```

File: app/templates/display.html [hunk 2]
```diff
@@ -35,7 +35,9 @@
 {{ response|safe }}
 </body>
 {% include 'footer.html' %}
-<script src="{{ cb_url('autocomplete.js') }}"></script>
+{% if autocomplete_enabled == '1' %}
+        <script src="{{ cb_url('autocomplete.js') }}"></script>
+    {% endif %}
```
Comment (benbusby) [inline app/templates/display.html:40]: Indentation issue
```suggestion
{% if autocomplete_enabled == '1' %}
    <script src="{{ cb_url('autocomplete.js') }}"></script>
{% endif %}
```

**Review by benbusby — CHANGES_REQUESTED** [review]
Looks good, just a couple minor formatting fixes before I merge.

File: app/routes.py [hunk 3]
```diff
@@ -37,6 +37,8 @@
 newest_version = '' if current_version >= newest_version \
     else newest_version
 
+ac_var = 'WHOOGLE_AUTOCOMPLETE'
+autocomplete_enabled = os.getenv(ac_var, '1')
```
Comment (benbusby) [inline app/routes.py:41]: pep8 also enforces 2 line breaks before function defs

```suggestion
autocomplete_enabled = os.getenv(ac_var, '1')

```

**Comment by DUOLabs333** [comment]
So, is it ready to be merged?

**Review by benbusby — APPROVED** [review]

**Pull request merged**